### PR TITLE
Fix #80: Update configuration parameters

### LINKED ISSUE #80: Update configuration parameters
Add configuration parameters:
- Token validity in PowerAuth server
- Proxy settings in PowerAuth server
- Recovery settings in PowerAuth server
- Proxy settings in Push server (update existing)
- FCM URL in Push server
- Android security warning in Web Flow
- Password encryption for Web Flow

Consider configuration of active Spring profiles (all projects).

Additionally, update:
- DDL scripts
- CSS
- Message resources
- Bootstrap version

### PR DESCRIPTION
**Pull request by romanstrobl**
Update of configuration, DDL and resources for 2019.05 release.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -95,6 +104,7 @@ PUSH_SERVER_CAMPAIGN_BATCH_SIZE=100000
 PUSH_SERVER_MESSAGE_STORAGE_ENABLED=false
 PUSH_SERVER_SPRING_BATCH_JOB_ENABLED=false
 PUSH_SERVER_SPRING_JMX_ENABLED=false
+PUSH_SERVER_SPRING_JMX_DEFAULT_DOMAIN=powerauth-push-server
```
Comment (saalistaja) [inline .env]: should be consistent with the name => `powerauth-push`

File: deploy/conf/default/powerauth-java-server.xml [hunk 2]
```diff
@@ -17,17 +17,34 @@
...
 
     <!-- Database Configuration - JNDI -->
     <Parameter name="spring.datasource.jndi-name" value="${POWERAUTH_SERVER_DATASOURCE_JNDI_NAME}"/>
 
     <!-- JMX Configuration -->
     <Parameter name="spring.jmx.enabled" value="${POWERAUTH_SERVER_SPRING_JMX_ENABLED}"/>
+    <Parameter name="spring.jmx.default-domain" value="${POWERAUTH_SERVER_SPRING_JMX_DEFAULT_DOMAIN}"/>
+
+    <!-- HTTP Proxy Settings -->
+    <Parameter name="powerauth.service.http.proxy.enabled" value="${POWERAUTH_SERVER_HTTP_PROXY_ENABLED}"/>
```
Comment (saalistaja) [inline deploy/conf/default/powerauth-java-server.xml]: I am still not sure if these should be called `POWERAUTH_SERVER` or `POWERAUTH_SERVICE`.

File: .env (hunk 1 above)
Comment (romanstrobl) [reply .env]: Currently we do not have any such project `powerauth-push`... we always use `powerauth-push-server`, it is also used in URLs, everyone knows it like this. I will keep it as is.

File: deploy/conf/default/powerauth-java-server.xml (hunk 2 above)
Comment (romanstrobl) [reply deploy/conf/default/powerauth-java-server.xml]: `SERVICE` is used in general for SOAP service. So I think it is fine.

File: .env (hunk 1 above)
Comment (saalistaja) [reply .env]: but the configuration three lines below is `PUSH_SERVER_APPLICATION_NAME=powerauth-push` I just like consistent things :)

Comment (romanstrobl) [reply .env]: Ah, ok, I will fix it.

**Pull request merged**